### PR TITLE
Updated contributing guide

### PR DESCRIPTION
**Pull request by nijotz**
## Proposed Release Notes
* Updated contributing guidelines

## Links
https://github.com/newrelic/newrelic-repo-template/blob/master/CONTRIBUTING.md

## Details
* Merged the New Relic contributing template with our existing contributing guidelines doc

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,38 +4,77 @@ New Relic welcomes code contributions by the Node community to this module, and
...
-## Testing
+### Testing
 
 The agent includes a suite of unit and functional tests which should be used to
 verify your changes don't break existing functionality.
 
 Unit tests are stored in `test/`. They're written in
-[Mocha](https://mochajs.org), and have the extension
-`.test.js`.
+[Mocha](https://mochajs.org), and have the extension `.test.js`.
```
Comment (michaelgoin) [inline CONTRIBUTING.md:64]: Technically, we are using `tap` now with some left behind using the mocha DSL until we finish converting.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,38 +4,77 @@ New Relic welcomes code contributions by the Node community to this module, and
...
 `.tap.js`.
 
-Functional tests against specific versions of instrumented modules are stored in
-`test/versioned/`. They are also written in `node-tap`.
+Functional tests against specific versions of instrumented modules are stored
+in `test/versioned/`. They are also written in `node-tap`.
 
-There are some other tests in `test/versioned-node/`. They are works in progress
-and not ready for general-purpose use.
+There are some other tests in `test/versioned-node/`. They are works in
```
Comment (michaelgoin) [inline CONTRIBUTING.md:73]: i don't think this exists anymore?

**Comment by astormnewrelic** [comment]
i signed it

File: CONTRIBUTING.md (hunk 1 above)
Comment (nijotz) [reply CONTRIBUTING.md]: Pushed an update with this removed

File: CONTRIBUTING.md (hunk 2 above)
Comment (nijotz) [reply CONTRIBUTING.md]: Correct! Removed

**Pull request merged**